README.md

```diff
@@ -1,3 +1,13 @@
 # Documentation
 
 everything you need to know about dahlia... WISDOM.
+
+### bootable usb 
+
+see **[here](https://github.com/HexaOneOfficial/documentation/blob/master/how%20to%20make%20a%20bootable%20usb.md/)** to make a bootable usb.
```

Comment:
please, update the link to point to the dahlia-os repository instead of your fork, i.e. https://github.com/dahlia-os/documentation/blob/master/how%20to%20make%20a%20bootable%20usb.md/